app/assets/javascripts/discourse/app/templates/password-reset.hbs

```diff
@@ -70,6 +70,10 @@
           {{/unless}}
         {{else}}
           <h2>{{i18n "user.change_password.choose"}}</h2>
+          {{#if this.errorMessage}}
+            <div class="alert alert-error">{{this.errorMessage}}</div>
+            <br />
+          {{/if}}
```

Comment:
Ensures we will show any error on the password step, like a 429, this is not ideal because atm 429 on the password step will show a rate limiting error related to OTP.